tasks/configure.yml

```diff
@@ -2,11 +2,13 @@
 
 # Manage Collectd configuration
 
+
+
 - name: 'CONFIG | MAIN | Manage collectd main configuration file'
   become: True
   template:
     src: 'etc/collectd/collectd.conf.j2'
```

Comment:
Now, I prefix all roles paths by {{ role_path }}, Can you update this statement please